plugin/src/main/java/at/helpch/chatchat/util/Kyorifier.java

```diff
@@ -0,0 +1,75 @@
+package at.helpch.chatchat.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+
+public final class Kyorifier {
+    private static final Map<Character, String> COLOURS = new HashMap<>(15);
+    private static final Map<Character, String> FORMATTERS = new HashMap<>(6);
+    private static final Pattern pattern = Pattern.compile("&(?<code>[\\da-fk-or])|[&{\\[<]?[#x](?<hex>(&?[A-Fa-f\\d]){6})[}\\]>]?");
```

Comment:
pattern should be renamed. just looking at the field name i have no idea what that does. 
and UPPER_CASE for static final fields pls

Reply:
yes yes of course. Just copy pasted it from the Kyorify expansion :)) Will make these changes.